frame/support/src/storage/bounded_vec.rs

```diff
@@ -225,7 +230,8 @@ impl<T, S: Get<u32>> BoundedVec<T, S> {
 
 	/// Get the bound of the type in `usize`.
 	pub fn bound() -> usize {
-		S::get() as usize
+		let bound: usize = S::get().into().try_into().unwrap();
```

Comment:
you can never `unwrap` in the runtime, this needs to be made into a safe conversion.

You can go to `u32` and then since wasm is a 32 bit system, you can do `as usize`.